pandas/tseries/tests/test_resample.py

```diff
@@ -2839,6 +2839,32 @@ def test_aggregate_with_nat(self):
             # if NaT is included, 'var', 'std', 'mean', 'first','last'
             # and 'nth' doesn't work yet
 
+    def test_resample_with_nat(self):
```

Comment:
Would it be helpful here? All three index types seem to handle NaT differently (see examples above).